restaurants_recommendation/reviews/models.py

```diff
@@ -1,16 +1,20 @@
 from django.db import models
+from django.db.models import Q
 
 
 class Review(models.Model):
-    rating = models.FloatField(null=True)
-    content = models.CharField(max_length=128)
+    rating = models.FloatField(default=0)
```

Comment:
constarints 부분이 값지정하기 위하여 설정한 것으로 확인됩니다!
field범위 내에서 아래와 같은 방법도 있을 것 같습니다 :) 

```suggestion
from django.core.validators import MaxValueValidator, MinValueValidator

rating = models.FloatField(
    validators=[MinValueValidator(0.0), MaxValueValidator(5.0)],
)
```

Reply:
좋은 정보 감사합니다. 
테스트를 해보니 validator를 사용하면 필드 자체에 값을 검사하기 때문에 serializer에서 rating에 대한 validation을 따로 수행하지 않아도 되네요!
다만 validator를 구성하면 python level에서만 동작하는데, 실무에서는 Database에 어느정도까지 제약을 추가하는지가 궁금하네요!

Reply:
평점이 중요한 것에 따라 다를 것 같습니다 :)